modules/nw-about-configuring-master-interface-container.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+
+[id="nw-about-configuring-master-interface-container_{context}"]
+= About configuring the master interface in the container network namespace
+
+In {product-title} 4.14 and later, the ability to allow users to create a MAC-VLAN, IP-VLAN, and VLAN subinterface based on a master interface in a container namespace is now generally available.
+
+This feature allows you to create the master interfaces as part of the pod network configuration in a separate network attachment definition. You can then base the VLAN, MACVLAN, or IPVLAN on this interface without requiring the knowledge of the network configuration of the node.
+
+To ensure the use of a container namespace master interface specify the `linkInContainer` and set the value to `true` in the VLAN, MACVLAN, or IPVLAN plugin configuration depending on the particular type of additional network.
+
+An example use case for utilizing this feature is to create multiple VLANs based on SR-IOV VFs. To do so, begin by creating an SR-IOV network and then define the network attachments for the VLAN interfaces.
+
+The following example shows how to configure the setup illustrated in this diagram.
+
+.Creating VLANs
+image::345_OpenShift_config_additional_network_0823.png[Creating VLANs]
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create a dedicated container namespace where you want to deploy your pods by using the following command:
++
+[source,terminal]
+----
+$ oc new-project test-namespace
+----
+. Create an SR-IOV node policy:
+
+.. Create an `SriovNetworkNodePolicy` object, and then save the YAML in the `sriov-node-network-policy.yaml` file:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
Shoud we mentioned that this paritucal configuration works for mlx only because of:
 deviceType: netdevice
 isRdma: false

Reply:
I added note up front.